### DISCUSSION TITLE
Guided Install Failed - x509 certificate signed by unknown authority

### DISCUSSION THREAD

**Discussion by wcmgto** [okd-project/okd#1586] [General]
I'm running the automated install on my own provisioned VMWare cluster following this guide - https://docs.okd.io/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#prerequisites

I'm at the point of running the installer after adding the VCenter trusted root CA to the client I'm running the install from - https://docs.okd.io/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-adding-vcenter-root-certificates_installing-vsphere-installer-provisioned

It appears the cert was added successfully because I can now navigate the vCenter URL without a cert warning. However, I get the following error when running the create cluster command. 

Fatal failed to fetch metadata... unable to connect to vCenter... ensure provided information is correct and client certs have been added to the system trust... x509: certificate signed by unknown authority. 

Any ideas of what the issue might be?

**Comment by jcpowermac** [comment]
@wcmgto that is usually all that is needed. What OS are you running the installer on?

**Reply by jcpowermac** [reply]
and please attach the full installer log, thanks!

**Reply by wcmgto** [reply]
I was using Ubuntu, but I just tried a Fedora system instead as that is the OS the commands reference in the official documentation. It worked fine in Fedora so it must be something specific to Ubuntu. Would like to figure out what the issue is.

**Reply by jcpowermac** [reply]
I don't use Ubuntu but I would figure just copying the ca from vcenter like mentioned in the ubuntu doc _should_ work.
https://ubuntu.com/server/docs/security-trust-store ?

**Reply by wcmgto** [reply]
Yeah, that was my thought and it's what I did. I could see that it imported successfully because I was able to browse to the vCenter URL without the cert warning so not sure why it was still having an issue.

**Reply by jcpowermac** [reply]
you could try 

```
export SSL_CERT_FILE=/path/to/ca
```

We use that in CI.
